modules/ROOT/partials/configuration/ai_request.adoc

```diff
@@ -0,0 +1,176 @@
+[[ai_request]]
```

Comment:
@MitchCTiny Here is the fiddle created based on this code snippet. I entered valid api key and AI request is made but nothing happening. No errors in console. Please check.

Here is the fiddle: https://fiddle.tiny.cloud/hDiaab

@MitchCTiny @tiny-ben-tran @brian-forte

Reply:
Another thing I found when I didn't provide an API key. The UI is stuck showing the loading icon because fetch-event-source keeps retrying if there is an 401 error

Reply:
@ShiridiGandham , we found two typos in the demo. Basically, `onMessage` and `onError` should be `onmessage` and `onerror` respectively.

Reply:
Api key - commented out
<img width="694" alt="image" src="https://github.com/tinymce/tinymce-docs/assets/43625458/d9354001-a332-4915-8c05-0f81226c1bc1">

Wrong api_key
<img width="697" alt="image" src="https://github.com/tinymce/tinymce-docs/assets/43625458/decce029-2dbd-4eb9-8ffc-acd97913afa1">

here is a working fiddle - need to update api key 
https://fiddle.tiny.cloud/rDiaab

LGTM